core/src/main/java/com/meltmedia/cadmium/core/meta/SiteConfigProcessor.java

```diff
@@ -49,14 +49,10 @@ public void processDir(String contentDirectory) throws Exception {
             processor.processFromDirectory(metaDir);
           } catch(IOException e) {
```

Comment:
We are catching exception and rethrowing the same exception .  Line: 50 and 52 are not needed.